src/components/webcams/Mjpegstreamer.vue

```diff
@@ -73,6 +73,12 @@ export default class Mjpegstreamer extends Mixins(BaseMixin, WebcamMixin) {
         return this.currentFPS < 10 ? '0' + this.currentFPS.toString() : this.currentFPS
     }
 
+    get showFpsCounter() {
+        if (!this.showFps) return false
+
+        return this.camSettings.extra_data?.hideFps ?? true
```

Comment:
invert bool. setting is hide fps, function is show
```suggestion
        return !this.camSettings.extra_data?.hideFps ?? true
```

Reply:
fixed with the new commit